lib/adhearsion-ivr/ivr_controller.rb

```diff
@@ -119,14 +127,18 @@ def on_failure(&block)
 
     def run
       @errors = 0
-      deliver_prompt
+      deliver_prompt true
```

Comment:
Happy to do so using keyword arguments, meaning 2.0.0+. What do you mean "CS discussion we had last week"?